From 69b4dd268c99651ca48779c2f4fc7bf0508e6abd Mon Sep 17 00:00:00 2001
From: thongsyho <thong.ho.px@rvc.renesas.com>
Date: Fri, 22 Jun 2018 17:29:35 +0700
Subject: [PATCH 252/628] drm: rcar-du: Fix help of DRM_RCAR_LVDS config

drm: rcar-du: Fix help of DRM_RCAR_LVDS config

Since DRM_RCAR_LVDS config is available for not only internal encoder
of R8A7790 but also external encoder, this help is fixed.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Hung Dong <hung.dong.xd@rvc.renesas.com>
Signed-off-by: Nguyen Van Linh [FGA.BU13] <LinhNv25@fsoft.com.vn>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
---
 drivers/gpu/drm/rcar-du/Kconfig | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/Kconfig b/drivers/gpu/drm/rcar-du/Kconfig
index 9bceba1..dba9131 100644
--- a/drivers/gpu/drm/rcar-du/Kconfig
+++ b/drivers/gpu/drm/rcar-du/Kconfig
@@ -24,8 +24,10 @@ config DRM_RCAR_LVDS
 	select DRM_PANEL
 	depends on ARCH_R8A7790 || ARCH_R8A7791 || COMPILE_TEST
 	help
-	  Enable support for the R-Car Display Unit embedded LVDS encoders
-	  (currently only on R8A7790 and R8A7791).
+	  Enable support LVDS encoders.
+
+	  If you want to access LVDS encoder, you select y.
+	  If you don't want to access LVDS encoder, you select n.
 
 config DRM_RCAR_VSP
        bool "R-Car DU VSP Compositor Support"
-- 
2.7.4

